### PR TITLE
Fix/activity view filters

### PR DESCRIPTION
**Pull request by Draym**

### CODE REVIEW

File: components/trade-activities/TradeActivitiesTable.tsx [hunk 1]
```diff
@@ -175,7 +177,6 @@ export function TradeActivitiesTable({
         {mergedActivities?.length ? (
           mergedActivities.map((activity, index) => (
             <ActivityRow
-              key={getActivityId(activity)}
```
Comment (raphael-verdier) [inline components/trade-activities/TradeActivitiesTable.tsx]: Why did you delete this?

**Review by raphael-verdier — APPROVED** [review]

File: components/trade-activities/TradeActivitiesTable.tsx (hunk 1 above)
Comment (Draym) [reply components/trade-activities/TradeActivitiesTable.tsx]: not used in ActivityRow (not even declared as a parameter). i used as 'id' for testing, but I removed it for the commit, because it wasn't in the HTML in the first place
